Source/Diffable/FetchedDiffableItem.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  FetchedDiffableItem.swift
+//  FetchedDataSource
+//
+//  Created by Jonathan Provo on 14/03/2022.
+//
+
+import Foundation
+
+public struct FetchedDiffableItem: Hashable {
+	public let sectionIdentifier: String
+	let object: NSObject
```

Comment:
Could we make this property public? Sometimes you want to check what actual type is contained by the item but don't use its actual value.